pingcap/tidb
# Makefile: make native gotest running in log_level=fatal

Signed-off-by: Shuaipeng Yu <jackysp@gmail.com>



### What problem does this PR solve? 
Native gotest is usually used for developers. If `make gotest` failed, it always prints lots of error logs which make it not easy to find the failed test cases.

### What is changed and how it works?
Change the log_level to fatal in native gotest.

### Check List 

Tests 

 - Manual test (add detailed scripts or steps below)
1. make a failed test
1. `make gotest`
1. check the output
